modules/icon-theme/hm.nix

```
@@ -0,0 +1,121 @@
+{ pkgs, lib, config, ... }:
+
+let
+  cfg = config.stylix.targets.iconTheme;
+in
+{
+  options.stylix.targets.iconTheme = {
+    enable = config.lib.stylix.mkEnableTarget "iconTheme" true;
+    name = lib.mkOption {
+      description = "The icon theme name";
+      type = lib.types.str;
+      default = "Numix-Circle";
+    };
+    package = lib.mkOption {
+      description = "The icon theme package";
+      type = lib.types.package;
+      default = pkgs.numix-icon-theme-circle;
+    };
+    adjustColorScheme = {
```

Maybe a shorter name for this option would be better?

```suggestion
    recolor = {
```